README.md

```diff
@@ -46,15 +39,15 @@ and drop it on your lxc templates path (usually `/usr/share/lxc/templates`)._
 ```sh
 git clone https://github.com/fgrehm/vagrant-lxc-base-boxes.git
 cd vagrant-lxc-base-boxes
-make precise
+make DEBIAN_BOXES=jessie debian
```

Comment:
This hunk seems unrelated to the actual commit of removing the 'unmaintained' notice from the README. Also I am not sure this change is appropriate.